lighthouse-core/test/audits/byte-efficiency/bundle-duplication-test.js

```diff
@@ -0,0 +1,224 @@
+              12650,
+            ],
+            "totalBytes": 0,
+            "url": "",
+            "urls": Array [
+              "https://example.com/coursehero-bundle-1.js",
+              "https://example.com/coursehero-bundle-2.js",
+            ],
+            "wastedBytes": 12650,
+          },
+          Object {
+            "source": "js/src/search/results/view/filter/autocomplete-list.tsx",
+            "sourceBytes": Array [
+              1143,
+              1134,
+            ],
+            "totalBytes": 0,
+            "url": "",
+            "urls": Array [
+              "https://example.com/coursehero-bundle-2.js",
+              "https://example.com/coursehero-bundle-1.js",
+            ],
+            "wastedBytes": 1134,
+          },
+          Object {
+            "source": "js/src/search/results/view/filter/autocomplete-filter.tsx",
+            "sourceBytes": Array [
+              3823,
+              3812,
+            ],
+            "totalBytes": 0,
+            "url": "",
+            "urls": Array [
+              "https://example.com/coursehero-bundle-1.js",
+              "https://example.com/coursehero-bundle-2.js",
+            ],
+            "wastedBytes": 3812,
+          },
+          Object {
+            "source": "js/src/search/results/view/filter/autocomplete-filter-with-icon.tsx",
+            "sourceBytes": Array [
+              2696,
+              2693,
+            ],
+            "totalBytes": 0,
+            "url": "",
+            "urls": Array [
+              "https://example.com/coursehero-bundle-1.js",
+              "https://example.com/coursehero-bundle-2.js",
+            ],
+            "wastedBytes": 2693,
+          },
+          Object {
+            "source": "js/src/common/component/school-search.tsx",
+            "sourceBytes": Array [
+              5840,
+              5316,
+            ],
+            "totalBytes": 0,
+            "url": "",
+            "urls": Array [
+              "https://example.com/coursehero-bundle-2.js",
+              "https://example.com/coursehero-bundle-1.js",
+            ],
+            "wastedBytes": 5316,
+          },
+          Object {
+            "source": "js/src/common/component/search/abstract-taxonomy-search.tsx",
+            "sourceBytes": Array [
+              3103,
+              3098,
+            ],
+            "totalBytes": 0,
+            "url": "",
+            "urls": Array [
+              "https://example.com/coursehero-bundle-1.js",
+              "https://example.com/coursehero-bundle-2.js",
+            ],
+            "wastedBytes": 3098,
+          },
+          Object {
+            "source": "Other",
+            "sourceBytes": Array [],
+            "totalBytes": 0,
+            "url": "",
+            "urls": Array [
+              "https://example.com/coursehero-bundle-1.js",
+              "https://example.com/coursehero-bundle-2.js",
+            ],
+            "wastedBytes": 10519,
```

Comment:
example of the folder thing I'd *love* to see `node_modules/@babel/runtime` and `node_modules/lodash-es` split out as rows :)

Reply:
ah, here's the clarification i needed

you are suggesting aggregating all the modules matching `node_modules/(@.*)?/(.*)/**/*` into one row?

Reply:
yeah exactly 👍

Reply:
I have concerns about doing that - consider lodash. One bundle might need `lodash/deep`, another `lodash/get`. Pretend each is pretty big. We'd then be falsely reporting that there's duplication across those two bundles. Using different modules from the same package in different bundles should be allowed.

Also, I just think it's simpler to keep this in terms of modules.

Reply:
Maybe there's a misunderstanding then. We wouldn't be falsely reporting any duplicates with what I'm proposing. This is exclusively for UI presentation in how we aggregate the already identified duplicates modules.

In the scenario that both `lodash/deep` and `lodash/get` are duplicated in different bundles, it seems pretty reasonable to say that "lodash" is duplicated. It at least gives the correct actionable advice. But the one real world situation we're analyzing here is even way more clear cut. Right now we have the completely unactionable "Other" when we could *at least* pull out `@babel/runtime` and `lodash-es`

Reply:
Got it! I'm cool with this - even though it loses some amount specificity, knowing exactly which modules are duplicated should be possible with the treemap viz I have planned for duplicated modules. so let's do this.